# MotorHighArray.msg - 25x高速数据数组
# 电机高速信息数组
MotorHigh[] motors      # 电机高速信息数